### DISCUSSION TITLE
支持通过抓取的请求链接获取抽卡记录

### DISCUSSION THREAD

**Discussion by YouZhengChuan** [sunfkny/genshin-gacha-export#45] [功能建议 / Ideas]
有部分玩家（例如我）只玩手机端原神，能不能支持一下，通过手机https抓包程序抓到的抽卡记录链接，来获取游戏的抽卡记录呢？包括武器池，角色池，常驻池。

**Comment by sunfkny** [comment]
其实是支持的， https://github.com/sunfkny/genshin-gacha-export/commit/ad66c8e9363b1f4d9b83955ca43f6db28478b490 这次更新用os.path.join代替了硬编码分隔符，甚至可以直接在手机上运行，参考 [配置文件](https://github.com/sunfkny/genshin-gacha-export/wiki/%E9%85%8D%E7%BD%AE%E6%96%87%E4%BB%B6)  打开`手动输入url `即可
如果是用termux的话浏览器打开抽卡报告都是可以适配的（目前没有合并进来）

**Reply by sunfkny** [reply]
删掉 requirements.txt 内 mitmproxy 和 pyinstaller 两行就行了，手机抓包获取链接不需要安装（我实测也装不上）
适配termux打开网页的方法是修改`render_html.py`末尾
```
    webbrowser.register("termux", None, webbrowser.GenericBrowser("termux-open")) # 注册 termux 打开网页
    webbrowser.open_new_tab("gachaReport.html")
```
